packages/api/src/middlewares/withFilters.ts

```diff
@@ -87,6 +89,20 @@ export function hasCustomFilters(filters: Filters) {
   );
 }
 
+export function getRollupFromAddressFilter(
```

Comment:
This function already exists in packages/api/src/utils/count.ts

Is it a mistake?

Reply:
Also, a merge-related error